### PR TITLE
Add Github workflow

### PR DESCRIPTION
**Pull request by perlpunk**

### CODE REVIEW

File: .github/workflows/ci.yaml [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+  workflow_dispatch:
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v2
+
+    - run: echo Hello, world!
```
Comment (ingydotnet) [inline .github/workflows/ci.yaml:20]: Oh hi there.

**Review by ingydotnet — CHANGES_REQUESTED** [review]
Thanks. Please fix the comments and also you can yank travis.

**Comment by perlpunk** [comment]
updated

**Review by ingydotnet — APPROVED** [review]
Nice! Thank you!!

**Pull request merged**